dbt/adapters/athena/session.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+import boto3.session
+from dbt.contracts.connection import Connection
+
+
+__BOTO3_SESSION__: Optional[boto3.session.Session] = None
+
+
+def get_boto3_session(connection: Connection = None) -> boto3.session.Session:
+    def init_session():
+        global __BOTO3_SESSION__
+        __BOTO3_SESSION__ = boto3.session.Session(
+            region_name=connection.credentials.region_name,
+            profile_name=connection.credentials.aws_profile_name,
```

Comment:
I'm not sure about this. What if I don't want to pass the aws profile? e.g. Running dbt transformations from ecs containers?

Reply:
Yes it will fallback to None, so I don't think it changes anything from before on that case.